modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.tsx

```diff
@@ -27,6 +28,10 @@ export function RightSidebarObjectRelationshipDetails({
 }: RightSidebarObjectRelationshipDetailsProps) {
 	const [{elements}] = useObjectFolderContext();
 	const [readOnly, setReadOnly] = useState(true);
+	const [
```

Comment:
Hi Nara! Its because on this file we only have this modal, so it didn't make sense to me to use showModals here; but I can change if you think its better!

Reply:
I followed the behavior we had in another files, whenever we have only one modal there, we just create a new state for it instead of using showModals; do you want me to change in these other files too?

Reply:
I changed on my file, let me know if you want me to send a follow-up changing in all other places!

Reply:
@rebsilva If this is already a pattern, you don't need to change it :)